### ISSUE TITLE
Vertx does not show up on current and outdated commands

### ISSUE THREAD

**Issue by marcoVermeulen** [gvmtool/gvm-cli#348]
Related to #345, arrays are not handled correctly and this results in truncation. Vertx is last in the list so drops off.

**Comment by marcoVermeulen** [comment]
@programmist I had a closer look on my system, and I seem to have vertx showing up on the current command. Could you try having another look, maybe after opening a new terminal?

**Comment by programmist** [comment]
I closed my terminal emulator and opened it up again.  When I run `gvm current` I see
```
Using:
gradle: 2.5
groovy: 2.4.4
```

When I run `gvm current vertx` I see
```
Using vertx version 3.0.0
```
I'm still new to gvm, but is there a local cache that might need to be cleared?  I tried `gvm flush candidates` and opened a new terminal, but it didn't change anything.

I also tried uninstalling gvm by deleting the .gvm directory and removing the entries from .bashrc, .bash_profile, and .zshrc.  Upon reinstalling gvm and Groovy, Gradle, and Vert.x I still don't see vertx in the current list.

I'm running OSX 10.10.4
